cqrs-example/debian.build-js.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -e
+sudo apt-get install npm nodejs-legacy
+cd static/
+npm install browserify minifyify babelify react classnames reflux
```

Comment:
Oh, I see I've somewhat, umm, 'under-documented' the build procedure for the front-end.

(Probably because I've been experimenting with a different non-JS frontend.)